src/sparseml/pytorch/optim/manager.py

```diff
@@ -266,10 +266,11 @@ def from_yaml(
              yaml str is also supported in place of a file path.
         :param add_modifiers: additional modifiers that should be added to the
             returned manager alongside the ones loaded from the recipe file
-        :param recipe_variables: additional variable values to override the recipe
-            with (i.e. num_epochs, init_lr)
+        :param recipe_variables: additional arguments to override any root variables
+            in the recipe with (i.e. num_epochs, init_lr)
         :return: ScheduledModifierManager() created from the recipe file
         """
+        recipe_variables = parse_recipe_variables(recipe_variables)
```

Comment:
can we get this added to the keras and tf managers

Reply:
done